isis/tests/data/chan1m32isis/badFile/bad.file

```diff
@@ -0,0 +1 @@
+>:(
```

Comment:
Is this needed or could we use an ofstream to create it programmatically? An example of this is in the campt tests.